src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import useFetch from './hooks/useFetch'
+
+const ArticlePage = () => {
+  const { id } = useParams()
+  const { data, loading, error } = useFetch(`https://be-nc-news-93e4.onrender.com/api/articles/${id}`)
+
+  if (loading) {
+    return <p>Loading...</p>
+  }
+  if (error) {
+    return <p>Error: {error.message}</p>
```

Comment:
This error message would return a message from your backend repo and be whatever you set it to there, just ensure whatever comes back is useful and readable to a user
Double check this for each error